pkgs/README.md

```diff
@@ -23,7 +23,20 @@ See the [CONTRIBUTING.md](../CONTRIBUTING.md) document for more general informat
 
 ## Quick Start to Adding a Package
 
-To add a package to Nixpkgs:
+Each new package has a cost for the maintainers, Hydra and users downloading
+Nixpkgs. Before adding a new package, please consider the following questions:
+* Is the package ready for general use? For example, if the author(s) is/are working towards the first release, it is probably too early.
+* Does the project have a license?
+* How realistic is it that it will be used by other people?
+* Are you going to maintain the package?
+
+If any of these questions' answer is no, then you should probably not add the
```

Comment:
Offer alternatives here IMHO: "consider a NUR, etc."

Reply:
fixed in 7fae21eb99c81c2f3998db54509c6c81f6063003